windows/benji.py

```diff
@@ -84,6 +84,16 @@ def events(frame,put):
 		cam.release()
 		cv2.destroyAllWindows()
 	
+	#Get lyrics
+	elif link[0] == "lyrics":
+		link = '+'.join(link[1:])
+		link = link.replace('+',' ')
+		title = link[1:]
+		goog_search = "https://www.google.com/search?sclient=psy-ab&client=ubuntu&hs=k5b&channel=fs&biw=1366&bih=648&noj=1&q=" + title + "lyrics"

```

Comment:
Do we have a better way of finding the lyrics than scrapping google result? 
Also, client=ubuntu is not a constant parameter.